README.md

```diff
@@ -0,0 +1,27 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> In this project I:
+- Set up linters for HTML and CSS.
+- I created an HTML file represent the project.
+- I style it using a CSS file
+
+
+## Built With
+
+- HTML
+- CSS
+- VSCODE
+
+## Live Demo
+
+Not available yet
+
+
+## Authors
+
+👤 Noureddine MEZIANY
+
+- GitHub: [@nmeziany](https://github.com/nmeziany)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/noureddine-meziany/)
```

Comment:
You have done amazing work✔️. Please use the suggested ReadMe [template](https://github.com/microverseinc/readme-template) to add what is missing in your Readme ( Contributing, Show your support, Acknowledgments, License).

Reply:
Changes done! 💯